devices/pine64-pinephone/kernel/default.nix

```diff
@@ -22,6 +23,10 @@ mobile-nixos.kernel-builder {
       url = "https://github.com/mobile-nixos/linux/commit/372597b5449b7e21ad59dba0842091f4f1ed34b2.patch";
       sha256 = "1lca3fdmx2wglplp47z2d1030bgcidaf1fhbnfvkfwk3fj3grixc";
     })
+    (fetchurl {
+      url = "https://github.com/dreemurrs-embedded/Pine64-Arch/raw/master/PKGBUILDS/pine64/linux-megi/pp-keyboard.patch";
```

Comment:
using `master` isn't safe, I think you should pin to a specific revision otherwise a futur build _may_ download a different content and the sha hash won't match.

Reply:
Oops, I swear I remember clicking permalink :smiling_face_with_tear: Fixed.